frontend/lib/identification/identification_qr_content_parser.dart

```diff
@@ -86,24 +75,19 @@ class IdentificationQrContentParser {
           unixExpirationDate == null) {
         return QRCodeParseResult(
```

Comment:
Well, it probably won't surprise you that I still am no fan of this error handling. Especially now when we think about i18n. It smells pretty fishy that we then would have to add translations to this parser class for example, and not to the UI class.

Suggestion: we could add as example a `ParserException` class. One could then e.g. add a subclass `ExpirationDateMissingParserException` and throw an instance of that here instead of returning a `QRCodeParserResult` with `error == true` and the german error message. One could then catch `ExpirationDateMissingParserException` in the UI class separately and show the message, if we want to inform the user about what is missing in detail, or just catch all `ParserException`s at once, display a generic warning to the user and still print a nice log message with details.

Reply:
@elKei24 Do you think this is better? https://github.com/ehrenamtskarte/ehrenamtskarte/blob/222-verification-backend-request/frontend/lib/verification/scanner/verification_qr_content_parser.dart

I haven't changed anything in the identification parser yet

Reply:
Well, the UI message is still in the same file, but I find it cleaner now 👍🏽